docs/CONTRIBUTING.md

```diff
@@ -38,12 +48,79 @@ template will be requested to be updated. Any pull requests opened without a
 corresponding issue may be delayed or be required to undergo further changes
 before being accepted.
 
-### Issues
+#### Pull Request Workflow
 
-All issues should follow our [ISSUE_TEMPLATE](ISSUE_TEMPLATE.md). It is
-additionaly recommended to prefix your issue with an appriopriate "tag" such as
-`[QUESTION]`, `[BUG]`, `[REGRESSION]`, `[IDEA]`, or `[PROPOSAL]`.
+As workflow for Pull Requests (PR) it is recommended to 
 
-### Questions
+1. Fork the [repository](https://github.com/terrafx/terrafx.interop.windows) 
+Make sure the git configuration has the upstream master set, 
+e.g. run `git remote add upstream https://github.com/terrafx/terrafx.interop.windows`
 
-Feel free to open an issue prefixed with `[QUESTION]`.
+2. Create a branch with name reflecting the issue/proposal you are planning to work on.
+
+3. Implement your PR
+
+4. git Commit, Fetch --all, Rebase, Push, and then submit your PR. 
+The submit can be done via git commandline, or from your git gui tool 
+or with the button on the github page of your fork. 
+
+5. If on your own or prompted by a PR review you make further changes, 
+then you can update your PR via Commit, Fetch --all, Rebase, Push. 
+The push will be automatically recognized by github and github 
+will update the PR on the main upstream repository.
+
+6. While one PR is in the works like this and maybe you are waiting on 
+a review you can make another branch for another PR and work on that in parallel.
+
+7. When a PR is approved and closed your code becomes part of the upstream code base 
+and there is no need to keep the branch. 
+Feel free to delete the branch to reduce clutter in your git log, 
+but this step is not required.
+
+#### Coding Conventions
+
+TerraFX follows the industry standard 
```

Comment:
This isn't quite true. I deviate from the Visual Studio defaults on a few places and there isn't necessarily an "industry standard" but rather recommended practices.

Reply:
But it is true that the .editor config sets the coding style to what you like? So if I resolve all warnings then it should be correct?

Reply:
The `.editorconfig` specifies the repo conventions yes; but there are a few issues with it that cause it to deviate.

In some cases there are also other `.editorconfig` files which specify different rules. For example, with the actual interop bindings they are meant to match the native conventions, so you can't really define anything that "always matches"

Reply:
ok.  Added an note that .editorconfig has limitations.